install_config/router/default_haproxy_router.adoc

```diff
@@ -648,6 +648,88 @@ $ oc scale dc/router --replicas=0 && oc scale dc/router --replicas=1
 ----
 
 
+[[finding-router-hostname]]
+== Finding the Hostname of the Router
+
+When the user exposes a service, the host name in the route is the DNS name that external
+users will use to access the application. The network admin for the external network must
+make sure the host name resolves to the name of a router that has admitted the route.
+This can be done by setting up a DNS CNAME record to point to the host name of the router.
+
+The host name of the router may not be know to the user. When it is not, the cluster administrator
+will provide it.  As a convenience, The cluster admin can provide the *--router-canonical-hostname* when
+creating the router. The *--routeri-canonical-hostname* is a string that contains the external
+canonical hostname of the router.
+
+For routers that already exist, the cluster admin can edit the router's dc and add the information
+in the *ROUTER_CANONICAL_HOSTNAME* environment variable.  The current *ROUTER_iCANONICAL_HOSTNAME*
+is displayed in the route status for all routers that have admitted the route.
+
+====
+----
+# oadm router myrouter --router-canonical-hostname="rtr.example.com"
+----
+====
+
+The above command ends up creating an environemnt variable *ROUTER_CANONCAL_HOSTNAME* in the
+Router's dc. *ROUTER_CANONCAL_HOSTNAME* contains the host name of the router.
+
+The user can run *oc get route -o yaml* to show the route status. The route status
+includes a list of routers that have admitted the route. The user selects a router from the
+list and finds the host name of the router to pass along to the network admin.
+
+====
+----
+$ oc get dc/myrouter -o yaml
+...
+spec:
+  template:
+    spec:
+      containers:
+        - env:
+          - name: ROUTER_CANONCAL_HOSTNAME
+            value: rtr.example.com
+----
+====
+
+When the user creates a route all of the active routers evaluate the route and, if conditions
+are met, admit it.  When a router that defines the *ROUTER_CANONCAL_HOSTNAME* environment variable admits
+the route the router places the value in the routerHostname field in the route status.  The user
+can examine the route status to determine which, if any, routers have admitted the route.
+
+There are two ways of displaying the routerHostname, oc get route -o yaml and oc describe route.
+
+====
+----
+$ oc get route hello-route3 -o yaml
+...
+status:
+  ingress:
+
+    conditions:
+      lastTransitionTime: 2016-12-07T15:20:57Z
+      status: "True"
+      type: Admitted
+      host: hello.in.mycloud.com
+      routerCanonicalHostname: rtr.example.com
+      routerName: myrouter
+      wildcardPolicy: None
+----
+====
+
+====
+----
+$ oc describe route/hello-route3
+...
+Requested Host: hello.in.mycloud.com exposed on router myroute (host rtr.example.com) 12 minutes ago
+----
+====
+
+With the above information in hand, the user can ask the network admin to have traffic to
```

Comment:
Maybe not network admin, but DNS admin?  And ask them to set up a CNAME from ... to ...